ui/desktop/app/styles/app.scss

```diff
@@ -695,7 +695,7 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     border-radius: 8px;
```

Comment:
I just realized this but because there's a scroll bar on the right, the border radius only visually shows for the left side
<img width="1686" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/53ab5e4d-49c7-46fb-872f-c07dc3c8d2dd">

Reply:
I am not seeing that locally. Are you using this branch and seeing that?

Reply:
Yup, I notice it whether we use canvas or not